### ISSUE TITLE
twentytwenty: course information div misaligned

### ISSUE THREAD

**Issue by eri-trabiccolo** [gocodebox/lifterlms#986]
### Reproduction Steps

+ visit a course, with course information block, using the twentytwenty theme

### Expected Behavior

+ I expect to see all the blocks aligned to the center

### Actual Behavior

+ the course information block is aligned to the left

![Schermata 2019-11-13 alle 16 50 23](https://user-images.githubusercontent.com/7689242/68780126-134c7580-0636-11ea-8341-ee5d67e73d4f.png)

This issue has been recently introduced with this commit in 2020:
https://github.com/WordPress/twentytwenty/commit/fe07e338deb797f6824e359202c1f884ffbdc370#diff-d778367bc83d62decbe66370245be94a

1 day before the release...
Basically with changes in that commit, the following CSS rule is not overridden:
https://github.com/gocodebox/lifterlms/blob/914f43d5857348216022c3e838cfbe265410fd0d/assets/scss/frontend/_course.scss#L2-L3
(left,right margin set to 0)
hence the misplacement.

### Error Messages / Logs
n/a

### System Information

WP 5.3
Twenty Twenty 1.0
LifterLMS 3.37.0

### Browser, Device, and Operating System Information
n/a
